man/systemd.resource-control.xml

```diff
@@ -1705,7 +1685,17 @@ DeviceAllow=/dev/loop-control
           <varname>BlockIOWriteBandwidth=<replaceable>device</replaceable> <replaceable>bytes</replaceable></varname>.
           Please switch to the unified cgroup hierarchy.</para>
 
- <xi:include href="version-info.xml" xpointer="v252"/></listitem>
+          <xi:include href="version-info.xml" xpointer="v252"/></listitem>
```

Comment:
Not your fault, but I do not think we should add version info for history section, as the version is already mentioned at the top.